CGPs/cgp-0044.md

```diff
@@ -3,9 +3,9 @@ cgp: 44
 title: Increasing Mento Bucket Sizes for cUSD and cEUR
 date-created: 2021-10-28
 author: '@martinvol, @rcroessmann'
-status: EXECUTED
+status: REJECTED
 discussions-to: https://forum.celo.org/t/economic-parameters-bucket-sizes-and-reserve-allocation-cgp-40-cgp-43/1997
-governance-proposal-id: 55
+governance-proposal-id: 45
```

Comment:
This was the original attempt which failed. CGP 051 was the re-attempt with proposal 55 that succeeded